src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -50,7 +51,7 @@ public class HelpWindow extends UiPart<Stage> {
             + "- Format: exit\n"
             + "- Example: exit";
     private static final String HELP_MORE_INFORMATION = "For more detailed information, "
-            + "- visit the PROperty User Guide at: https://ay2425s1-cs2103t-f15-3.github.io/tp/UserGuide.html";
+            + " visit the PROperty User Guide at: https://ay2425s1-cs2103t-f15-3.github.io/tp/UserGuide.html";
```

Comment:
There is an extra whitespace at the beginning of the sentence! You might want to remove it before merging.